02-technical-guides/hierarchies/01-hierarchy-fundamentals.ipynb

```diff
@@ -0,0 +1,1592 @@
+{
```

Comment:
<p>Same comment as on the other notebook, for consistency's sake, can we use "alias" instead of "set variables" language in this comment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/890/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>This is fixed, thanks for catching this!</p>